packages/sitecore-jss-manifest/src/generator/generate.ts

```diff
@@ -164,10 +163,26 @@ export interface GenerateToFileOptions extends GenerateOptions {
   excludeMedia?: boolean;
 }
 
-// generates the JSON manifest and returns it as a variable
-// NOTE: media is not copied into the manifest when using this method,
-// and no files are written to disk. Use generateToFile() to make a manifest
-// that is designed to get packaged/imported.
+/**
+ * generates the JSON manifest and returns it as a variable
+ * NOTE: media is not copied into the manifest when using this method,
+ * and no files are written to disk. Use generateToFile() to make a manifest
+ * that is designed to get packaged/imported.
+ *
+ * @param {Object} config
+ * @param {string} config.requireArg
+ * @param {string[]} config.fileGlobs
+ * @param {any} config.pipelines
+ * @param {string} config.appName
+ * @param {boolean} [config.excludeItems]
+ * @param {boolean} [config.excludeDictionary]
+ * @param {string} config.language
+ * @param {string[]} config.pipelinePatchFileGlobs
+ * @param {boolean} config.debug
+ * @param {boolean} config.wipe
+ * @param {string[]} config.rootPlaceholders
+ * @param {boolean} config.skipPlaceholderBlacklist
```

Comment:
Looks like the JSDoc plugin got a little confused here with the fix.
```suggestion
 * @param {GenerateOptions} options
```

Reply:
Changed